packages/embedded/src/main/java/elide/embedded/interop/NativeAppCallbackHolder.java

```diff
@@ -0,0 +1,40 @@
+package elide.embedded.interop;
+
```

Comment:
missing license headers in many places but these an be added before next release. spotless _may_ add them if you run it, but i'm not sure automatic license headers are working yet. intellij adds them most of the time for me.

Reply:
i'll have intellij add the headers now, forgot about that